react/Checkbox/Readme.md

```diff
@@ -17,3 +17,9 @@
 ```
 <div><Checkbox label="This is a mixed checkbox" mixed /></div>
 ```
+
+### Checkbox with complicated children
```

Comment:
```suggestion
### Checkbox with complex children
```

https://english.stackexchange.com/questions/10459/what-is-the-difference-between-complicated-and-complex